packages/cfgen/cfgen.1.0.0-preview.6/opam

```diff
@@ -0,0 +1,46 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Generate AWS CloudFormation templates from code"
+description: """
+This project provides the
+  ability to generate AWS CloudFormation templates from OCaml code, with a support library to ease its use.
+  AWS resource definitions have been used to generate constructs for building CloudFormation resources
+  in a type-safe way."""
+maintainer: ["Chris Armstrong <oss@chrisarmstrong.dev>"]
+authors: ["Chris Armstrong"]
+license: "MIT"
+homepage: "https://github.com/chris-armstrong/ocaml-cfgen"
+bug-reports: "https://github.com/chris-armstrong/ocaml-cfgen/issues"
+depends: [
+  "dune" {>= "3.4" & >= "3.4"}
```

Comment:
```suggestion
  "dune" {>= "3.4"}
```

Although that's "as generated by dune" if we believe the header. So maybe it has to do with the content of the dune-project file? Or something else?

It's not critical to remove anyway.

Reply:
I added an explicit dune dependency, but it looks like it's generated this more unusual one? I am generating from a dune-project file, how do i fix this?